Project-Hotpot/Project-Hotpot.xcodeproj/xcuserdata/harinisundaram.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -7,7 +7,7 @@
 		<key>Project-Hotpot.xcscheme_^#shared#^_</key>
 		<dict>
 			<key>orderHint</key>
-			<integer>0</integer>
+			<integer>4</integer>
```

Comment:
Consider adding a .gitignore file (like this one: https://github.com/github/gitignore/blob/main/Swift.gitignore) to reduce the noise from folders like xcuserdata.